Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/README.md

```diff
@@ -0,0 +1,53 @@
+SecneurX provides real-time threat intelligence that protects companies against the latest cyber threats, including APTs, phishing, malware, ransomware, data exfiltration, and brand infringement. Security teams rely on our dependable and rich data to expand their threat landscape visibility, resulting in improved detection rates and response times.
+This integration was integrated and tested with version 1.0.0 of SecneurX Threat Feeds
+
+## Configure SecneurX Threat Feeds on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for SecneurX Threat Feeds.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API Key | Input the API key for fetching feed from the source. | True |
+    | Fetch indicators | Select this option if you want this integration instance to fetch indicators from the SecneurX Threat Intelligence feed. | False |
```

Comment:
Please regenerate the README as is missed the updated params like `url` `proxy` etc

Reply:
Will update this as well.